content/en/blog/news/2023-10-joining-the-cncf.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Joining the CNCF Sandbox"
+linkTitle: "Joining the CNCF Sandbox"
+date: 2023-10-20
+---
+
+A year after kcp.io launched as the project website, we have big news to share. kcp is joining the CNCF and becomes a CNCF Sandbox project. A huge thanks to the sponsors of [our application](https://github.com/cncf/sandbox/issues/47). We are excited to join an amazing community of open source projects!
+
+If you haven't checked in with the kcp project in a while, let us bring you up to speed on the last few months. In April this year, Red Hat [announced](https://groups.google.com/g/kcp-users/c/yTe8RMcpxrg) that kcp won't have its immediate focus and extended an invitation to the community to take over the project.
+
+A group of committed organizations and individuals decided to respond to the call for action and established a [new project governance](https://github.com/kcp-dev/kcp/blob/main/GOVERNANCE.md) and bootstrapped a [new list of maintainers](https://github.com/kcp-dev/kcp/pull/2953) to lead kcp as a community project. Since then, more people have joined the list of maintainers to drive the kcp project forward. We relaunched kcp.io in a new design recently to commemorate the "new" kcp project.
+
+From the inception of the new project governance, it was clear that kcp does not exist in a vacuum. It is a citizen of the Kubernetes ecosystem and as such, joining the CNCF to formalize that citizenship was the obvious and natural road to take. Thankfully, after hearing our case, the [Technical Oversight Committee (TOC)](https://www.cncf.io/people/technical-oversight-committee/) decided to accept us into the CNCF Sandbox program.
```

Comment:
```suggestion
From the inception of the new project governance, it was clear that kcp does not exist in a vacuum. It is a citizen of the Kubernetes ecosystem and as such, joining the CNCF to formalize that citizenship was the obvious and natural road to take. Eventually, after evaluating our application, the [Technical Oversight Committee (TOC)](https://www.cncf.io/people/technical-oversight-committee/) decided to accept us into the CNCF Sandbox program.
```

Reply:
"Thankfully, after hearing our case" sounds a little like "they saved us" 😀 Might be only in my non-native ears.

Reply:
That's fair - "hearing our case" is not the best way to say this, thinking about it.